metadata-ingestion/tests/integration/sql_server/golden_files/procedures/DemoData.Foo.NewProc.json

```diff
@@ -26,6 +26,7 @@
                     "downstreams": [
                         "urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:mssql,demodata.foo.age_dist,PROD),age)"
                     ],
+                    "transformOperation": "COPY: [persons].[age] AS [age]",
```

Comment:
Is this correct with the square brackets?

Reply:
yes - mssql uses square brackets for quotes